html/audio-worklet.js

```diff
@@ -543,6 +537,7 @@ class Player extends AudioWorkletProcessor {
       this.local_latency = 150 * sampleRate / 1000;  // rough initial guess (150ms)
       this.click_volume = 0;
       this.input_gain = 1.0;
+      this.time_callbacks = [];
     })
```

Comment:
It looks to me like this is always stored in time-sorted order, and you maintain that when inserting. Could you add a comment about this invariant?

Reply:
Belatedly fixed in https://github.com/gwillen/solstice-audio-test/pull/125 .